ExpenseManagerTransactions/Controllers/TransactionsController.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExpenseManagerTransactions.DomainModels;
+using ExpenseManagerTransactions.RequestModels;
+using ExpenseManagerTransactions.Services;
+using ExpenseManagerTransactions.Utils;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExpenseManagerTransactions.Configurations
+{
+    [ApiController]
+    [Authorize]
+    [Route("api/transaction")]
+    public class TransactionsController : ControllerBase
+    {
+        private readonly ITransactionService _transactionService;
+
+        public TransactionsController(ITransactionService transactionService)
+        {
+            _transactionService = transactionService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAllTransactions()
+        {
+            IEnumerable<Transaction> transactions = await _transactionService.GetAllTransactions(ClaimsExtensions.GetUserID(User));

```

Comment:
At some point, we need to make this endpoint return some user info in order to show the data sync from user service to transaction service.